data/human/culture conversations.txt

```diff
@@ -103,6 +103,35 @@ mission "Trash Fire Crops"
 
 
 
+mission "Worker's Occasion"
+	minor
+	source
+		government "Republic"
+		attributes "dirt belt"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`Pausing to consider a restaurant for your dinner, you notice a local worker staring at its menu, hand shaking as he walked in. His clothes are neatly worn, and look freshly brushed, but it's impossible not to notice the dirty marks and stains that just haven't washed out. The man is undeniably nervous and twitchy, yet no one kicks him out. Looking at the menu yourself, you realize the prices are very affordable for you but you recall how unreachable they would've been for you before your career change.`
```

Comment:
> Pausing to consider a restaurant for your dinner, you notice a local worker staring at its menu, ...

At first glance i thought the "its" was referring to the worker. "the menu" or a more elaborate version would work just as well.

> hand shaking as he walked in.

Bit odd since you just noticed him staring, yet you're supposed to have seen him walk in?

Reply:
The pitfalls of hastily proofreading a draft. XD
```suggestion
			`Pausing to consider a restaurant for your dinner, you notice a local worker staring at the menu, hand shaking as he proceeds to walk in. His clothes are neatly worn, and look freshly brushed, but it's impossible not to notice the dirty marks and stains that just haven't washed out. The man is undeniably nervous and twitchy, yet no one kicks him out. Looking at the menu yourself, you realize the prices are very affordable for you but you recall how unreachable they would've been for you before your career change.`
```